mssw/Animal/lib/LivingCreature.pm

```diff
@@ -17,77 +19,171 @@ Version 0.01
 our $VERSION = '0.01';
 
 =head1 SYNOPSIS
-
-Quick summary of what the module does.
-
-Perhaps a little code snippet.
-
-    use LivingCreature;
-
-    my $foo = LivingCreature->new();
-    ...
-
+#
```

Comment:
I suspect this was @Spinodal's way of getting codio syntax highlighting readable with perldoc.